### ISSUE TITLE
Not able to build page locally

### ISSUE THREAD

**Issue by eriksven** [eclipse-sdv-blueprints/blueprints-website#3]
Hello @nacidai, what versions of node and Docusaurus did you use to build the website?

I am tried to build it on my machine and keep getting errors like the following. Do you have an idea what I might do wrong?

```bash
[ERROR] Unable to build website for locale en.
[ERROR] Error: The docs folder does not exist for version "current". A docs folder is expected to be found at docs.
    at getVersionMetadataPaths (/Users/jes1be/Documents/SDV/GIT/blueprints/blueprints-website/node_modules/@docusaurus/plugin-content-docs/lib/versions/files.js:123:15)
    at async createVersionMetadata (/Users/jes1be/Documents/SDV/GIT/blueprints/blueprints-website/node_modules/@docusaurus/plugin-content-docs/lib/versions/index.js:97:68)
    at async Promise.all (index 0)
    at async readVersionsMetadata (/Users/jes1be/Documents/SDV/GIT/blueprints/blueprints-website/node_modules/@docusaurus/plugin-content-docs/lib/versions/index.js:150:30)
    at async Object.pluginContentDocs [as plugin] (/Users/jes1be/Documents/SDV/GIT/blueprints/blueprints-website/node_modules/@docusaurus/plugin-content-docs/lib/index.js:32:30)
    at async initializePlugin (/Users/jes1be/Documents/SDV/GIT/blueprints/blueprints-website/node_modules/@docusaurus/core/lib/server/plugins/init.js:83:32)
    at async Promise.all (index 0)
    at async initPlugins (/Users/jes1be/Documents/SDV/GIT/blueprints/blueprints-website/node_modules/@docusaurus/core/lib/server/plugins/init.js:91:21)
    at async loadPlugins (/Users/jes1be/Documents/SDV/GIT/blueprints/blueprints-website/node_modules/@docusaurus/core/lib/server/plugins/index.js:26:21)
    at async load (/Users/jes1be/Documents/SDV/GIT/blueprints/blueprints-website/node_modules/@docusaurus/core/lib/server/index.js:76:58)
[INFO] Docusaurus version: 2.4.3
Node version: v18.20.4
```

**Comment by nacidai** [comment]
You need to create an empty docs folder in the root folder prior to a build or run locally

**Comment by eriksven** [comment]
Thanks, I will try that.

**Comment by eriksven** [comment]
Thanks, that seemed to have worked. I will create a PR to add this hint to the Readme and the close this issue.

**Comment by eriksven** [comment]
will be closed by #4

**Comment by eriksven** [comment]
#4 has been merged
